Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -1,41 +1,19 @@
-# Eye Tracking Examples in MRTK
-This page covers how to get quickly started with using Eye Tracking in MRTK by building on our provided MRTK example package.
-The samples let you experience one of our new magical input capabilities: **Eye Tracking**! 
-The demo include a number of different use cases for seamlessly combining information about what you are looking at with **Voice** and **Hand** input. 
-This enables users to quickly and effortlessly select and move holograms across their view simply by looking at a target and saying _'Select'_ or performing a hand gesture. 
+# Eye tracking examples in MRTK
+This page covers how to get quickly started with using eye tracking in MRTK by building on our provided [MRTK eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking).
+The samples let you experience one of our new magical input capabilities: **Eye tracking**! 
+The demo includes a number of different use cases ranging from implicit eye-based activations to how to seamlessly combine information about what you are looking at with **voice** and **hand** input. 
+This enables users to quickly and effortlessly select and move holographic content across their view simply by looking at a target and saying _'Select'_ or performing a hand gesture. 
 The demos also include an example for eye-gaze-directed scroll, pan and zoom of text and images on a slate. 
 Finally, an example is provided for recording and visualizing the user's visual attention on a 2D slate.
+In the following, we will go into more detail what each of the different samples in the [MRTK eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking) includes:
 
+![List of eye tracking scenes](../Images/EyeTracking/mrtk_et_list_et_scenes.jpg)
 
-## Overview of MRTK Eye Tracking Samples
+We will start with a quick overview of what the individual eye tracking demo scenes are about.
+The individual sample scenes will be [loaded additively](https://docs.unity3d.com/ScriptReference/SceneManagement.LoadSceneMode.Additive.html).
+In case, you wonder what that entails, we will explain further below how to set up and test the MRTK eye tracking demos.
```

Comment:
Nit: This sentence doesn't really add that much.

Initially I thought that this sentence would tell me more about additive scene loading ("you wonder what this entails") but then the second half of the sentence tells a totally different story (we'll explain how to test the demos!)

It reminds me of something someone told me the other day when they said "today is a cloudy day, as a result I want chocolate"

Reply:
It's not completely unrelated. There have been many questions of the additive loading and I wanted to raise it early on that 1) Yes, we do additive scene loading and 2) If you don't know what that means (because a lot of people didn't) then don't worry, we'll get to it. 
But anyways... I rephrased it for now, but it's already outdated anyways with the new scene loading system.